internal/kafka/avail_status_msg.go

```diff
@@ -40,10 +41,10 @@ func genericMessage(ctx context.Context, m any, key string, topic string) (Gener
 		Value: payload,
 		// Keep headers written in lowercase to match sources comparison.
 		Headers: GenericHeaders(
-			"content-type", "application/json",
-			"x-rh-identity", identity.IdentityHeader(ctx),
```

Comment:
Consistency of what exactly? Header names? We are not consistent indeed, HTTP headers [are not case sensitive](https://stackoverflow.com/questions/5258977/are-http-headers-case-sensitive) and it is common to use CamelCase for them. However, TIL that for HTTP/2 the lowercase is actually enforced. This will be done by the HTTP/2 client, but I will use lowercase from now on. Let me fix thi.